core/geometry/src/geometry3d/Matrix3d.ts

```diff
@@ -2389,14 +2422,30 @@ export class Matrix3d implements BeJSONFunctions {
       if (z < 0.0)
         result.scaleColumnsInPlace(1.0, -1.0, -1.0);
     } else {
+      // if coordinate is (x,y,0), i.e., Front or Back or Left or Right view
+      /**
+       * The matrix that the "else" statement creates is
+       *        [-s   -s1*c    c1*c]
+       *        [c    -s1*s    c1*s]
+       *        [0      c1      s1 ]
+       * where
+       *        c = x / sqrt(x*x + y*y)
+       *        s = y / sqrt(x*x + y*y)
+       *        c1 = sqrt(x*x + y*y) / sqrt(x*x + y*y + z*z)
+       *        s1 = z / sqrt(x*x + y*y + z*z)
+       *
+       * This is an orthogonal matrix meaning it rotates the standard XYZ axis to ABC axis system
+       * (if matrix is [A B C]). The matrix rotates (0,0,1), i.e., the default Top view or Z axis,
+       *  to the eye point (x/r,y/r,z/r). The matrix also rotates (1,0,0) to a point on XY plane.
+       */
```

Comment:
Consider moving this comment block inside the `(z !== 0.0)` block further below, which is where it actually applies.